### ISSUE TITLE
Syndicate/ling ID runtime.

### ISSUE THREAD

**Issue by Zuhayr** [PolarisSS13/Polaris#384]
Occurs at roundstart.
````
runtime error: type mismatch: += the chitinous card (/obj/item/weapon/card/id/syndicate/changeling)
proc name: New (/obj/item/weapon/card/id/syndicate/New)
  source file: cards_ids_syndicate.dm,13
  usr: null
  src: the chitinous card (/obj/item/weapon/card/id/syndicate/changeling)
  call stack:
the chitinous card (/obj/item/weapon/card/id/syndicate/changeling): New(null)
the chitinous card (/obj/item/weapon/card/id/syndicate/changeling): New(null)
world: ()
## ERROR: File not found (config/config.txt)
## ERROR: File not found (config/game_options.txt)
## ERROR: File not found (config/dbconfig.txt)
## ERROR: File not found (config/forumdbconfig.txt)
````

Bonus round:
````
## WARNING: The empty jar attempted to add a reagent called 'slime' which doesn't exist. ()
````

**Comment by Neerti** [comment]
Chitinous card is my bad.  No idea what's wrong with the jar.

**Comment by Kelenius** [comment]
Fixed by #850
